cmake/projects/double-conversion/hunter.cmake

```diff
@@ -0,0 +1,22 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_configuration_types)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    double-conversion
+    VERSION
+    "3.0.0"
+    URL
+    "https://github.com/google/double-conversion/archive/v3.0.0.tar.gz"
+    SHA1
+    d900d4d946beb493a03cd6c9b180ed2bcbd0f20e
+)
+
+hunter_configuration_types(double-conversion CONFIGURATION_TYPES Release)
```

Comment:
What is the reason of this one? Default configuration is Debug + Release, user can set his own configuration in local config (see https://github.com/ruslo/hunter/wiki/example.hunter_configuration_types).

Reply:
I saw some other libraries do this. Thought it was a standard approach for low level libraries (like this one).

Reply:
I saw some other libraries do this. Thought it was a standard approach for low level libraries (like this one).

Reply:
> I saw some other libraries do this

They do it for other reasons. Usually because of the bugs and limitations.